### DISCUSSION TITLE
Standardnotes slow startup on Ubuntu 22.04 Snap

### DISCUSSION THREAD

**Discussion by qwertyuiopzxcvbnmlkjhgfdsa1** [standardnotes/forum#2876] [General] [answered]
**Describe the bug**
I am using StandardNotes quite often as main source of my memory. Excellent app

What is really bad is startup of application. It takes 10+ second.  Sometimes i click twice, because it seems to be dead. And gnome desktop does not sign me if app is starting or not.

**To Reproduce**
1. On Ubuntu 22.04 Gnome(Wayland or Xorg) click on standard-notes
2. Wait and wait....THIS IS a BUG
3. App starts correctly

**Expected behavior**
Standard notes starts faster

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04 Snap
 - Version Snap latest

**Comment by effieeee** [comment]
Hi @qwertyuiopzxcvbnmlkjhgfdsa1, is this still the case on the latest version of the Snap? Do you also encounter similar behavior with the AppImage?

**Comment by qwertyuiopzxcvbnmlkjhgfdsa1** [comment, marked as answer]
For version standard-notes 3.23.206  it starts in around 2 seconds now. Maybe first time after update it was slower, but not sure.

From my perspective slow startup is not bug any more in snap

@effieeee I can test appimage startup speed, if you send a link

**Comment by effieeee** [comment]
I see, though yes, you may still test it out with the [AppImage](https://github.com/standardnotes/app/releases/tag/%40standardnotes%2Fdesktop%403.23.206) for the same version!
